src/einsteinpy/tests/test_metric/test_kerrnewman.py

```diff
@@ -162,3 +167,25 @@ def test_calculate_trajectory_iterator_RuntimeWarning():
         for _, _ in zip(range(1000), it):
             pass
         assert len(w) >= 1
+
+
+def test_event_horizon_for_nonrotating_uncharged_case():
+    M = 5e27
+    a = 0.0
+    Q = 0.0
+    _scr = schwarzschild_radius_dimensionless(M)
+    a1 = kerrnewman_utils.event_horizon(M, a, Q, np.pi / 4, "Spherical")
+    a2 = kerr_utils.event_horizon(M, a, np.pi / 4)
+    assert_allclose(a1, a2, rtol=0.0, atol=1e-4)
```

Comment:
The tests seems fine,
set `rtol=1e-4` and `atol=0` for every new test you added.

Reply:
Also move the tests to `test_utils/test_kerrnewman_utils.py`

Reply:
Last thing, take care of this `rtol`, `atol` business in the same tests of `test_utils/test_kerr_utils.py`

Reply:
okay, on it! :)